src/field.cpp

```diff
@@ -461,6 +461,28 @@ field_id field_from_ident(const std::string &field_ident)
     return fd_null;
 }
 
+void map::create_burnproducts( const tripoint p, const item &fuel ) {
+    std::vector<material_id> all_mats = fuel.made_of();
+    if ( all_mats.empty() ) { 
```

Comment:
Should be like `if( all_mats.empty() ) {`. Also excess space after curly bracket.